tests/lib/dependencies.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+
+# shellcheck source=tests/lib/pkgdb.sh
+. "$TESTSLIB/pkgdb.sh"
+
+export DEPENDENCY_PACKAGES=
+export DISTRO_BUILD_DEPS=
+
+add_pkgs(){
+  DEPENDENCY_PACKAGES="$DEPENDENCY_PACKAGES $@"
```

Comment:
We're trying to standardize on four spaces per indent on shell and yaml code. I've seen some other files which are already all over the place, with mixed 2, 3 and 4 spaces on the same file. Let's please fix that as we drive by these files (no need to rush it), and also try not to introduce new inconsistencies meanwhile.